test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/io/ReadersTest.scala

```diff
@@ -22,6 +22,21 @@ import org.junit.Assert._
 import org.scalajs.testsuite.utils.AssertThrows._
 
 /** Tests for our implementation of java.io._ reader classes */
+class ReaderTest {
+  object MyReader extends java.io.Reader {
+    def read(dbuf: Array[Char], off: Int, len: Int): Int = {
+      java.util.Arrays.fill(dbuf, off, off + len, 'A')
+      len
+    }
+    def close(): Unit = ()
+  }
+
+  @Test def skip_should_always_skip_n_if_possible(): Unit = {
+    assertEquals(42, MyReader.skip(42))
+    assertEquals(2147483700L, MyReader.skip(Int.MaxValue.toLong + 53))
```

Comment:
Apparently, this test takes way too much time on Rhino, which is why the CI failed. Please add the following at the beginning of the method to fix that:
```scala
assumeFalse("Too slow in Rhino", executingInRhino)
```
with the following imports at the top:
```scala
import org.junit.Assume._

import org.scalajs.testsuite.utils.Platform._
```

Reply:
Applied the changes in https://github.com/scala-js/scala-js/pull/3665/commits/d51ba432689c13b324e1914f00770bfa4aeea813

Reply:
Well, apparently this test is too slow, period. Not just with Rhino. I tried running the CI a few times on this PR, but it keeps failing.

So could you add one test with a skip greater than 8192 but still small (e.g., 10000) so that there is a test that exercises the loop, but remove the test with 2147483700L altogether (and then also remove the assumption and the corresponding imports)?

Thanks.

Reply:
I did that small change and force-pushed, so that I can merge this PR before going on vacation.